### PR TITLE
Allow non required tags to have a 'None' option in the dropdown

### PR DESCRIPTION
**Pull request by introwit**

### CODE REVIEW

File: app/Services/Html/Concerns/Forms.php [hunk 1]
```diff
@@ -38,13 +38,19 @@ public function date(string $name = '', ?string $value = ''): Input
             ->class('-datetime');
     }
 
-    public function category(string $type): Select
+    public function category(string $type, bool $required): Select
```
Comment (sebastiandedeyne) [inline app/Services/Html/Concerns/Forms.php]: Maybe default the new parameter to `true` so it behaves like before?

**Review by sebastiandedeyne — CHANGES_REQUESTED** [review]
Looks great! One little nitpick :)

**Comment by introwit** [comment]
@sebastiandedeyne done :smile:

**Comment by sebastiandedeyne** [comment]
🙆‍♂️

**Pull request merged**